css/globals.css

```diff
@@ -0,0 +1,661 @@
+  padding: 2px 20px 2px 20px;
+}
+
+#navbar .submenu-content a:hover {
+  background: var(--hover-bg);
+  text-decoration: none;
+}
+
+@media all and (max-width: 1110px) {
+  .submenu-content {
+    width: 100%;
+    position: static;
+    display: none;
+    margin-top: 7px;
+    background-color: var(--card-bg);
+    padding: 0;
+    margin: 0;
+    border: none;
+    border-radius: 0;
+    box-shadow: none;
+    max-height: 190px;
+    overflow-y: auto;
+    overflow-x: hidden;
+    transform: none;
+    cursor: pointer;
+  }
+
+  .submenu-content li>a {
+    width: 100%;
+  }
+
+  .submenu-content li:first-child a {
+    border-radius: 0;
+  }
+
+  .submenu-content li:last-child {
+    border-bottom: 0;
+  }
+
+  .submenu-content li:last-child a {
+    border-radius: 0;
+  }
+
+  .submenu-content.open {
+    display: inline-block;
+  }
+
+  .submenu-content li {
+    border-bottom: 1px solid var(--border);
+  }
+}
+
+
+/* -------- i18n Button --------*/
+
+#language-picker-menu {
+  display: none;
+}
+
+@media all and (max-width: 1110px) {
+  #language-picker-menu #navmenu>li:first-child {
+    display: flex;
+  }
+
+  #language-picker-menu #navmenu {
+    max-height: 70vh;
+    overflow-y: auto;
+    scrollbar-width: thin;
+    text-align: center;
+  }
+
+  #language-picker-menu {
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 100%;
+    z-index: 1000;
+  }
+}
+
+/* -------- theme button --------*/
+
+#theme-icon-container {
+  cursor: pointer;
+}
+
+@media all and (max-width: 1110px) {
+  #theme-icon-container {
+    z-index: 10000;
+  }
+}
+
+/* -------- Print --------*/
+
+@media print {
+  header {
+    position: absolute;
+  }
+
+  #mobile-menu {
+    display: none;
+  }
+}
+
+/* ----------------- Blog --------------------- */
+
+#blog-side-menu-container {
+  position: fixed;
+  margin: 0;
+  padding: 0 10px 0 0;
+  top: 153px;
+  left: 30px;
+  height: 500px;
+  text-align: left;
+  font-size: 13px;
+  overflow-y: auto;
+}
+
+#blog-side-menu-container {
+  max-width: 210px;
+}
+
+#blog-side-menu-container li {
+  max-width: 100%;
+  margin-bottom: 5px;
+}
+
+#blog-side-menu-container>h3>a {
+  color: var(--fg)
+}
+
+#blog-side-menu {
+  list-style: none;
+  padding: 0;
+}
+
+#blog-side-menu>li>a {
+  color: var(--menu-grey);
+  font-weight: bold;
+  font-size: 13px;
+}
+
+@media all and (max-width: 1110px) {
+  #blog-side-menu-container {
+    display: none;
+  }
+}
+
+/* ----------------- Side Menu --------------------- */
+
+#menu {
+  position: fixed;
+  margin: 0;
+  padding: 0 10px 0 0;
+  top: 153px;
+  left: 30px;
+  height: 500px;
+  text-align: left;
+  font-size: 13px;
+  overflow-y: auto;
+}
+
+#menu ul a.active {
+  color: var(--fg)
+}
+
+#menu em {
+  font-weight: bold;
+  color: var(--card-fg);
+}
+
+#menu li {
+  list-style: none;
+}
+
+#menu ul {
+  height: 0;
+  overflow: hidden;
+}
+
+#menu ul.active {
+  height: auto;
+  padding: 0;
+}
+
+#menu a {
+  color: var(--menu-grey);
```

Comment:
```suggestion
  color: var(--menu-grey);
  display: block;
  padding-block: 0.2rem;
  padding-inline: 0.2rem;
  margin-inline: 0.2rem;
  border-radius: 0.2rem;
  
  &:is(:hover, :focus) {
    color: var(--fg);
    outline: 1px solid var(--fg);
   }
```
increase clicking area for all links

![image](https://github.com/user-attachments/assets/442bea9b-8173-443d-9a6a-0c0d9cfb9921)

1st one is :focus and another is :hover, use same styles is common standard